platform.h

```diff
@@ -0,0 +1,62 @@
+#ifndef BOARD_H
+#define BOARD_H
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define LED_ON 0
+#define CAN_5V_ON 1
+#define CAN_12V_ON 1
+
+#define SAMPLE_FREQ (1000.0 / MAX_SENSOR_LOOP_TIME_DIFF_ms)
+#define LOW_PASS_ALPHA(TR) ((SAMPLE_FREQ * TR / 5.0) / (1 + SAMPLE_FREQ * TR / 5.0))
+#define LOW_PASS_RESPONSE_TIME 1 // seconds
+
+// Time between main loop code execution
+#define MAX_LOOP_TIME_DIFF_ms 500
+// Time between "high speed" sensor checks
+#define MAX_SENSOR_LOOP_TIME_DIFF_ms 25
+// Reset if we go this long without seeing any CAN messages (including our own)
+#define MAX_BUS_DEAD_TIME_ms 10000
+
+// Voltage monitoring in 100k Ohms
+#define BATT_RESISTANCE_DIVIDER 5.6
+#define GROUND_RESISTANCE_DIVIDER 5.6
```

Comment:
Not clear what these are or where they are used. Are CONVERSION_RATIO_BATT_VOLT and CONVERSION_RATIO_5V_VOLT not used to convert the voltage sense input?